### PR TITLE
Relationship/friendship page

### PR DESCRIPTION
**Pull request by imjeee**
Factored out the code for relationships from Application.java to Relationships.java, also added the friendship page.

### CODE REVIEW

File: app/models/User.java [hunk 1]
```diff
@@ -72,10 +72,10 @@
   public String password;
 
   @OneToMany(mappedBy = "from", cascade = CascadeType.ALL)
-  public List<Relationship> friends; // A list of the user's work history
+  public List<Relationship> friends; // A list of the user's friendship history
 
   @OneToMany(mappedBy = "to", cascade = CascadeType.ALL)
-  public List<Relationship> friendedBy; // A list of the user's work history
+  public List<Relationship> friendedBy; // A list of the user's friendship history
```
Comment (amshali) [inline app/models/User.java:43]: Why do we want this?

File: app/controllers/Application.java [hunk 2]
```diff
@@ -14,10 +14,10 @@ public static void about(Long id) {
     render(user);
   }
 
-  public static void news(Long id) {
-    User user = id == null ? user() : (User) User.findById(id);
-    render(user);
-  }
+	public static void news(Long id) {
```
Comment (jtessler) [inline app/controllers/Application.java]: I think you replaced the space indentation for tabs here.
We should probably keep the space indentation (as discussed in class and [here](https://github.com/w7cook/OpenBook/wiki/Coding-Convention)).

**Comment by jtessler** [comment]
You should probably commit a quick change to replace all your tab indentation with spaces. See more [here](https://github.com/w7cook/OpenBook/wiki/Coding-Convention).

File: app/models/User.java (hunk 1 above)
Comment (imjeee) [inline app/models/User.java:43]: This is code from before. I didn't want to mess with it. I suppose this isn't necessary. Should I take it out?

File: app/controllers/Application.java (hunk 2 above)
Comment (imjeee) [inline app/controllers/Application.java]: I just committed and push to my own origin master. Is there anything else I need to do? Appreciate the help.

Comment (jtessler) [inline app/controllers/Application.java]: Nope, that's all you need to do! Notice the "imjeee added some commits" at the bottom of the pull request.

Comment (imjeee) [inline app/controllers/Application.java]: sweet. Thanks!

File: app/models/User.java (hunk 1 above)
Comment (amshali) [inline app/models/User.java:43]: Yes please do. If we wanted it we could put it back.

Comment (amshali) [inline app/models/User.java:43]: drop me a note after you do it

Comment (imjeee) [inline app/models/User.java:43]: nvm. This is actually needed in checking friendship and in news.

File: app/views/Relationships/relationships.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#{extends 'main.html' /}
+All Friends:<br />
+#{list items:user.confirmedFriends(), as:'friendship'}
+<a href="@{Application.news(friendship.to.id)}">${ friendship.to.name }</a>  <a id="friend-button" href="@{Relationships.removeFriends(user.id)}">remove friend</a><br />
```
Comment (amshali) [inline app/views/Relationships/relationships.html:4]: When I click on the name, it throws an error. Test it first with bob@gmail.com

Comment (amshali) [inline app/views/Relationships/relationships.html:4]: did you fix this error?

Comment (imjeee) [inline app/views/Relationships/relationships.html:4]: I clicked around. I'm not sure if that's fixed at some point, but doesn't seem to have an error now.

**Pull request closed without merging**